jdaviz/configs/default/plugins/subset_plugin/subset_plugin.vue

```diff
@@ -112,11 +112,14 @@
       </v-row>
     </div>
 
-    <v-row v-if="!multiselect" justify="end">
-      <j-tooltip v-if="can_simplify" tooltipcontent="Convert composite subset to use only add mode to connect subregions">
-        <v-btn color="primary" text @click="simplify_subset">Simplify</v-btn>
-      </j-tooltip>
-      <v-btn color="primary" text @click="update_subset">Update</v-btn>
-    </v-row>
+      <v-row v-if="!multiselect" justify="end" no-gutters>
+        <j-tooltip v-if="can_freeze" tooltipcontent="Freeze subset to a mask on the underlying data entries">
+          <v-btn color="primary" text @click="freeze_subset">Freeze</v-btn>
+        </j-tooltip>
+        <j-tooltip v-if="can_simplify" tooltipcontent="Convert composite subset to use only add mode to connect subregions">
+          <v-btn color="primary" text @click="simplify_subset">Simplify</v-btn>
+        </j-tooltip>
```

Comment:
I wonder if we should always show the simplify button but just disable it when needed?  Freezing is more enabled/disabled as a _setting_, whereas a disappearing simplify button might be confusing? 🤷‍♂️ 

```suggestion
        <j-tooltip tooltipcontent="Convert composite subset to use only add mode to connect subregions">
          <v-btn :disabled="!can_simplify" color="primary" text @click="simplify_subset">Simplify</v-btn>
        </j-tooltip>
```